storage-proofs/src/zigzag_drgporep.rs

```diff
@@ -199,16 +199,14 @@ mod tests {
             // prove_verify_32_3_1(32, 3, 1);
             // prove_verify_32_3_2(32, 3, 2);
 
-           prove_verify_fixed_32_5_1(5, 1);
            prove_verify_fixed_32_5_2(5, 2);
-           prove_verify_fixed_32_5_3(5, 3);
+           prove_verify_fixed_32_5_4(5, 4);
```

Comment:
Changing these just to use an even expansion degree (don't know how the tests work or if this is fine to do).

Reply:
that is fine